docs/tutorials/Qiskit Algorithms Transition Guide.ipynb

```diff
@@ -0,0 +1,1629 @@
+    "```\n",
+    "See the documentation of the [deprecated Grover](https://qiskit.org/documentation/stubs/qiskit.aqua.algorithms.Grover.html#qiskit.aqua.algorithms.Grover) and [new Grover](#)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Transition guide\n",
+    "\n",
+    "For oracles provided as circuits and a `is_good_state` function to determine good states"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit.circuit import QuantumCircuit\n",
+    "\n",
+    "oracle = QuantumCircuit(2)\n",
+    "oracle.cz(0, 1)\n",
+    "\n",
+    "def is_good_state(bitstr):\n",
+    "    return sum(map(int, bitstr)) == 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Previously:**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/manoel/projects/Quantum/qiskit-aqua/qiskit/aqua/algorithms/amplitude_amplifiers/grover.py:216: DeprecationWarning: The package qiskit.aqua.algorithms.amplitude_amplifiers is deprecated. It was moved/refactored to qiskit.algorithms.amplitude_amplifiers (pip install qiskit-terra). For more information see <https://github.com/Qiskit/qiskit-aqua/blob/master/README.md#migration-guide>\n",
+      "  'qiskit.algorithms.amplitude_amplifiers', 'qiskit-terra')\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Top measurement: 11\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit.aqua.algorithms import Grover\n",
+    "\n",
+    "grover = Grover(oracle, is_good_state, quantum_instance=aqua_qinstance)\n",
+    "result = grover.run()\n",
+    "print('Top measurement:', result.top_measurement)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**New:**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Top measurement: 11\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit.algorithms import Grover, AmplificationProblem\n",
+    "\n",
+    "problem = AmplificationProblem(oracle=oracle, is_good_state=is_good_state)\n",
+    "grover = Grover(quantum_instance=qinstance)\n",
+    "result = grover.amplify(problem)\n",
+    "print('Top measurement:', result.top_measurement)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since we are streamling all algorithms to use the `QuantumCircuit` class as base primitive, defining oracles using the `qiskit.aqua.compontents.Oracle` class is deprecated. Instead of using e.g. the `LogicalExpressionOracle` you can now use the `PhaseOracle` circuit from the circuit library."
```

Comment:
```suggestion
    "Since we are streamlining all algorithms to use the `QuantumCircuit` class as base primitive, defining oracles using the `qiskit.aqua.compontents.Oracle` class is deprecated. Instead of using e.g. the `LogicalExpressionOracle` you can now use the `PhaseOracle` circuit from the circuit library."
```

Reply:
I fixed it. Thank you.